junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/descriptor/TestFactoryTestDescriptor.java

```diff
@@ -69,6 +70,18 @@ public boolean isLeaf() {
 	}
 
 	@Override
+	public JupiterEngineExecutionContext execute(JupiterEngineExecutionContext context,
```

Comment:
At the moment, no, I don't think we want to introduce any additional lifecycle annotations.